publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/dynamodbstream/ImportCandidateDynamoDbStreamToEventBridgeHandler.java

```diff
@@ -0,0 +1,134 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.IOException;
+import java.net.URI;
+import java.time.Instant;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Function;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.events.models.AwsEventBridgeDetail;
+import no.unit.nva.events.models.EventReference;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.events.bodies.ImportCandidateDataEntryUpdate;
+import no.unit.nva.publication.model.business.Entity;
+import no.unit.nva.publication.model.business.Resource;
+import no.unit.nva.publication.model.business.importcandidate.ImportCandidate;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.attempt.Failure;
+import nva.commons.core.exceptions.ExceptionUtils;
+import nva.commons.core.paths.UnixPath;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequest;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+import software.amazon.awssdk.services.s3.S3Client;
+
+public class ImportCandidateDynamoDbStreamToEventBridgeHandler
+    implements RequestHandler<DynamodbEvent, EventReference> {
+
+    public static final ImportCandidate NO_VALUE = null;
+    private static final String PROCESSING_EVENT_MESSAGE = "Processing event for identifier: {}";
+    private static final String DETAIL_TYPE_NOT_IMPORTANT = "See event topic";
+    private static final String EMITTED_EVENT_MESSAGE = "Emitted Event:{}";
+    private static final String DYNAMO_DB_STREAM_SOURCE = "DynamoDbStream";
+    private static final Logger logger = LoggerFactory.getLogger(
+        ImportCandidateDynamoDbStreamToEventBridgeHandler.class);
+    private final S3Driver s3Driver;
+    private final EventBridgeClient eventBridgeClient;
+
+    @JacocoGenerated
+    public ImportCandidateDynamoDbStreamToEventBridgeHandler() {
+        this(S3Driver.defaultS3Client().build(), defaultEventBridgeClient());
+    }
+
+    protected ImportCandidateDynamoDbStreamToEventBridgeHandler(S3Client s3Client,
+                                                                EventBridgeClient eventBridgeClient) {
+        this.s3Driver = new S3Driver(s3Client, EVENTS_BUCKET);
+        this.eventBridgeClient = eventBridgeClient;
+    }
+
+    @Override
+    public EventReference handleRequest(DynamodbEvent inputEvent, Context context) {
+        var dynamodbStreamRecord = inputEvent.getRecords().getFirst();
+        var dataEntryUpdateEvent = convertToUpdateEvent(dynamodbStreamRecord);
+        return sendEvent(dataEntryUpdateEvent, context);
+    }
+
+    private static SortableIdentifier getIdentifier(ImportCandidateDataEntryUpdate blobObject) {
+        return Optional.ofNullable(blobObject.getOldData())
```

Comment:
In `ImportCandidateDataEntryUpdate`, both `getOldData` and `getNewData` should return `Optional` so that the caller can handle the expressed uncertainty.